install/Makefile

```diff
@@ -155,6 +155,16 @@ endif
 check-admin: kubectl
 	@output=$$(kubectl get crd 2>&1) || (echo "****" && echo "**** ERROR: Cannot continue as user is not a Cluster-Admin ****" && echo "****"; exit 1)
 
+crd-api-support: kubectl
```

Comment:
What's the point in having the check for CRD API support in Makefile?  Does the task still have some value for developers outside of CI?  If it doesn't I think it rather makes more sense to have it in the set of GitHub CI scripts.